stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1180,7 +1180,8 @@ extension String {
   /// values found in the `String`.
   public
   func propertyListFromStringsFileFormat() -> [String : String] {
-    return _ns.propertyListFromStringsFileFormat() as! [String : String]
+    return _ns.propertyListFromStringsFileFormat()! as [NSObject : AnyObject]
+      as! [String : String]
```

Comment:
I see.  Yes, bridging conversions throw in a lot of unprincipled weirdness in the type system.